scripts/bash/run_enceladus.sh

```diff
@@ -399,6 +399,7 @@ if [[ "$DEPLOY_MODE" == "client" ]]; then
   ADDITIONAL_JVM_CONF="$ADDITIONAL_JVM_CONF_CLIENT"
 else
   ADDITIONAL_JVM_CONF="$ADDITIONAL_JVM_CONF_CLUSTER"
+  add_spark_conf_cmd "spark.yarn.submit.waitAppCompletion" "true"
```

Comment:
This is supposed to be false. You don't want to wait for it to complete. You want to detach.